src/Common/src/System/Security/Cryptography/DSACng.ImportExport.cs

```diff
@@ -326,7 +326,6 @@ public override DSAParameters ExportParameters(bool includePrivateParameters)
 
             private static int FromBigEndian(byte[] b)
             {
-                Debug.Assert(b.Length == 4);
```

Comment:
Why delete the assert? More than 4 is ignoring data, less than 4 is going to throw an exception.